### PR TITLE
Fix Rails/OutputSafety violations

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
Replaced `html_safe` and `raw` with `tag` + `safe_join`

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -282,16 +282,16 @@ def display_estimated_user_count?
     estimated_user_count > LARGE_USERBASE_THRESHOLD
   end
 
-  # rubocop:disable Rails/OutputSafety
   def admin_config_label(method, content = nil)
-    content ||= raw("<span>#{method.to_s.humanize}</span>")
+    content ||= tag.span method.to_s.humanize
```
Comment (rhymes) [inline app/helpers/application_helper.rb:286]: nitpick non blocking: I noticed there's inconsistency with parenthesis, `tag.span` it's called without it and `tag.a` up in the first file is called with it, and `tag.label` down here is called with them. I vote for parenthesis :D

**Review by rhymes — APPROVED** [review]
Yeah!

**Review by Ridhwana — APPROVED** [review]
Nice one @lightalloy!

**Review by citizen428 — APPROVED** [review]

**Comment by rhymes** [comment]
@lightalloy you can merge master now

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (lightalloy) [reply app/helpers/application_helper.rb]: Nice catch :)

**Pull request merged**